legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -141,6 +142,10 @@ def _get_businesses_eligible_query(exclude_in_dissolution=True):
                     _is_limited_restored(),
                     _is_xpro_from_nwpta()
                 )
+            ).\
+            order_by(
+                no_transition_filed_after_restoration.desc(),
+                oldest_restoration_filing_date.asc(),
```

Comment:
Just wondering, what will happen when there are businesses have the same effective date? then by filing id or random?